src/main/java/wtf/meier/tariff/interpreter/model/RentalPeriod.kt

```diff
@@ -0,0 +1,17 @@
+package wtf.meier.tariff.interpreter.model
+
+import wtf.meier.tariff.interpreter.model.rate.RateCalculator
+import wtf.meier.tariff.interpreter.extension.minus
+import java.time.Instant
+import java.util.concurrent.TimeUnit
+
+data class RentalPeriod(
+    val rentalStart: Instant = Instant.ofEpochMilli(0),
+    val rentalEnd: Instant,
+    val calculatedStart: Instant = rentalStart,
```

Comment:
Can you think of a more descriptive name here?

Reply:
What do you think about invoicedStart and invoicedEnd?

Reply:
Sounds good. Maybe we can add a little doc entry why we need this as it's not super obvious :) 👍